CMakeLists.txt

```diff
@@ -138,7 +138,7 @@ if (USE_PMEM)
     if(PKG_CONFIG_FOUND)
         pkg_check_modules(LIBPMEM REQUIRED libpmem>=1.0)
     else()
-        find_package(LIBPMEM REQUIRED 1.0)
+        find_package(LIBPMEM QUIET REQUIRED 1.0)
```

Comment:
why did you add quiet? this should fail.

Reply:
Done ( I removed QUIET)